SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -167,6 +167,7 @@ const Schema_FieldId PLAYER_SPAWNER_SPAWN_PLAYER_COMMAND_ID = 1;
 // AuthorityIntent codes and Field IDs.
 const Schema_FieldId AUTHORITY_INTENT_VIRTUAL_WORKER_ID					= 1;
 const VirtualWorkerId INVALID_VIRTUAL_WORKER_ID							= 0;
+const ActorLockToken INVALID_ACTOR_LOCK_TOKEN							= 0;
```

Comment:
It doesn't seem to be the right place for `INVALID_VIRTUAL_WORKER_ID` and `INVALID_ACTOR_LOCK_TOKEN`, since they're not Schema_FieldIds.

Reply:
Moving both entries further down the file out of the `Schema_FieldId` list